### PR TITLE
Adjust binary llmaj to new engines and add rits support

### PR DESCRIPTION
**Pull request by lilacheden**

### CODE REVIEW

**Review by yoavkatz — APPROVED** [review]

File: src/unitxt/llm_as_judge.py [hunk 1]
```diff
@@ -65,12 +65,12 @@ def verify(self):
             )
 
         if isinstance(self.inference_model, OpenAiInferenceEngine):
-            if self.format and type(self.format) is not SystemFormat:
-                raise ValueError(
-                    "Error in 'LLMAsJudge' metric. Inference model 'OpenAiInferenceEngine' does "
-                    "not support formatting. Please remove the format definition from the recipe"
-                    " (OpenAi Chat API take care of the formatting automatically)."
-                )
+            # if self.format and type(self.format) is not SystemFormat:
```
Comment (OfirArviv) [inline src/unitxt/llm_as_judge.py:68]: change it to be either None, formats.empty or formats.chat_api

**Review by OfirArviv — CHANGES_REQUESTED** [review]

File: src/unitxt/llm_as_judge.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yoavkatz) [reply src/unitxt/llm_as_judge.py:68]: Except if it's empty format , which is ok.

Comment (lilacheden) [reply src/unitxt/llm_as_judge.py:68]: @OfirArviv - the format is the artifact, not the string. How can I check the string from the artifacrt?

[the remark this answers is not in the document]
Comment (lilacheden) [reply src/unitxt/llm_as_judge.py:68]: @yoavkatz - the current check was an error anyway as it allows for any format besides chat_api. see the above discussion with Ofir

Comment (lilacheden) [reply src/unitxt/llm_as_judge.py:68]: @OfirArviv - to be more clear, I can check if the format is a chat api format instance, but don't know how to check if a system format is generated from "formats.empty" - this is probably where this bug emerged from in the first place

**Review by OfirArviv — APPROVED** [review]

**Pull request merged**